Documentation/microsoft-git.md

```diff
@@ -0,0 +1,39 @@
+Details about the Microsoft/Git Fork
+====================================
+
+The [`microsoft/git` fork of Git](https://github.com/microsoft/git)
+is used to take early performance patches or custom patches for
+[VFS for Git](https://github.com/microsoft/git).
+
+Integration Testing with C# Layer
+---------------------------------
+
+If you have a pull request in `microsoft/git`, then you
```

Comment:
maybe:  ... in `microsoft/git` that passes all the normal Git-specific CI tests, ...